### PR TITLE
[bug] Add 5 missing fields to Alerts schema

### LINKED ISSUE #3007: [BUG]  Alert suppression fields missing from alert schema table
Add alert suppression fields to the [alert schema table](https://www.elastic.co/guide/en/security/master/alert-schema.html):

<table>
<tr>
	<td> 8.x alert field
    <td> Description
	<td> Type
<tr>
	<td> kibana.alert.suppression.terms.field
	<td> The field(s) used to group alerts for suppression
	<td> keyword
<tr>
	<td> kibana.alert.suppression.terms.value
	<td> The value(s) in the suppression fields
	<td> keyword
<tr>
	<td> kibana.alert.suppression.start
	<td> The timestamp of the first document in the suppression group
	<td> date
<tr>
	<td> kibana.alert.suppression.end
	<td> The timestamp of the last document in the suppression group
	<td> date
<tr>
	<td> kibana.alert.suppression.docs_count
	<td> The number of suppressed alerts
	<td> long
</table>

cc: @marshallmain 

Make sure to backport this to `8.7` and `8.6`.

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3007 by adding missing fields to the Alerts schema

Preview: [Alert schema](https://security-docs_3008.docs-preview.app.elstc.co/guide/en/security/master/alert-schema.html)

### CODE REVIEW

File: docs/reference/alert-schema.asciidoc [hunk 1]
```diff
@@ -142,4 +142,19 @@ Type: flattened
 |N/A | `kibana.alert.rule.producer` | Type: keyword
 |N/A | `kibana.alert.rule.rule_type_id` | Type: keyword
 
+|N/A | `kibana.alert.suppression.terms.field` | The field(s) used to group alerts for suppression.
```
Comment (nastasha-solomon) [inline docs/reference/alert-schema.asciidoc:145]:
```suggestion
|N/A | `kibana.alert.suppression.terms.field` | The fields used to group alerts for suppression.
```

Comment (nastasha-solomon) [reply docs/reference/alert-schema.asciidoc:145]: I'm pretty sure we don't use the parenthesis "s" in our docs so I changed it to plural.

**Review by nastasha-solomon — APPROVED** [review]
Just two minor edits - LGTM otherwise!

File: docs/reference/alert-schema.asciidoc [hunk 2]
```diff
@@ -142,4 +142,19 @@ Type: flattened
...
+
+Type:	keyword
+|N/A | `kibana.alert.suppression.terms.value` | The value(s) in the suppression fields.
```
Comment (nastasha-solomon) [inline docs/reference/alert-schema.asciidoc:148]:
```suggestion
|N/A | `kibana.alert.suppression.terms.value` | The values in the suppression fields.
```

**Review by joepeeples — APPROVED** [review]

**Review by marshallmain — APPROVED** [review]

**Pull request merged**